lib/prolongator.cu

```diff
@@ -49,165 +50,152 @@ namespace quda {
 
   };
 
+  template <int fineSpin, int fineColor, int coarseSpin, int coarseColor>
+  struct enabled : std::false_type { };
+
+#ifdef NSPIN4
+  template <> struct enabled<4,  3, 2,  6> : std::true_type { };
+  template <> struct enabled<4,  3, 2, 24> : std::true_type { };
+  template <> struct enabled<4,  3, 2, 32> : std::true_type { };
+  template <> struct enabled<2, 24, 2, 32> : std::true_type { };
+  template <> struct enabled<2, 32, 2, 32> : std::true_type { };
+  template <> struct enabled<2,  6, 2,  6> : std::true_type { };
+#endif
+#ifdef NSPIN1
+  template <> struct enabled<1,  3, 2, 24> : std::true_type { };
+  template <> struct enabled<1,  3, 2, 64> : std::true_type { };
+  template <> struct enabled<1,  3, 2, 96> : std::true_type { };
+  template <> struct enabled<2, 24, 2, 64> : std::true_type { };
+  template <> struct enabled<2, 24, 2, 96> : std::true_type { };
+  template <> struct enabled<2, 64, 2, 64> : std::true_type { };
+  template <> struct enabled<2, 64, 2, 96> : std::true_type { };
```

Comment:
This is missing 96 -> 96

Reply:
2b195025a8fccffb71846155ac7c79de9816acf3